ssclj/jar/src/main/clojure/com/sixsq/slipstream/ssclj/resources/common/std_crud.clj

```diff
@@ -3,18 +3,27 @@
   (:require
     [clojure.walk                                             :as w]
     [clojure.pprint                                           :refer [pprint]]
+    [clojure.tools.logging                                    :as log]
+
     [com.sixsq.slipstream.ssclj.resources.common.authz        :as a]
     [com.sixsq.slipstream.ssclj.resources.common.utils        :as u]
     [com.sixsq.slipstream.ssclj.resources.common.crud         :as crud]
     [com.sixsq.slipstream.ssclj.db.impl                       :as db]
-    [com.sixsq.slipstream.ssclj.resources.common.debug-utils  :as du]
 
     [com.sixsq.slipstream.ssclj.resources.common.cimi-filter  :as cf]
     [com.sixsq.slipstream.ssclj.resources.common.pagination   :as pg]))
 
+(defn log-request
+  [request]
+  (log/info (:request-method request)
+            (:uri request)
+            (get request :query-string "")
+            (get request :body "")))
```

Comment:
@st Wondering if it's more useful to explicitly log when something was not found, instead of a blank string:

``` clojure
(get request :query-string "<no :query-string>")
(get request :body         "<no :body>")
```